src/technologydata/package_data/dea_energy_storage/dea_energy_storage.py

```diff
@@ -0,0 +1,656 @@
+# SPDX-FileCopyrightText: The technology-data authors
+#
+# SPDX-License-Identifier: MIT
+
+"""Data parser for the DEA energy storage data set."""
+
+import argparse
+import logging
+import pathlib
+import re
+import typing
+
+import pandas
```

Comment:
```suggestion
import pandas as pd
```

and use that in the script (for consistency, doc strings are using `pd.` already.